python1.py

```diff
@@ -0,0 +1,87 @@
+import random
+print("Choose a template:")
+print("1. Hospital Story")
+print("2. Camping Story")
+print("3. Castle Story")
+n = input("1, 2, 3: ")
+if (n == '1'):
+    Number = input("Number: ")
+    Measure_of_time = input("Measure_of_time: ")
+    Mode_of_Transportation = input("Mode_of_Transportation: ")
+    Adjective = input("Adjective: ")
+    Adjective2 = input("Adjective2: ")
+    Noun = input("Noun: ")
+    Color = input("Color: ")
+    Part_of_the_Body = input("Part_of_the_Body: ")
+    Verb = input("Verb: ")
+    Number2 = input("Number2: ")
+    Noun2 = input("Noun2: ")
+    Noun3 = input("Noun3: ")
+    Part_of_the_Body_2 = input("Part_of_the_Body_2: ")
+    Noun4 = input("Noun4: ")
+    Adjective3 = input("Adjective3: ")
+    Silly_Word = input("Silly_Word: ")
+    print("It was about", Number, Measure_of_time, " ago when I arrived at the hospital in a ", Mode_of_Transportation, ". ")
+    print("The hospital is a/an ", Adjective, " place, there are a lot of ", Adjective2, Noun, " here. ")
+    print("There are nurses here who have ", Color, Part_of_the_Body ,".")
+    print("If someone wants to come into my room I told them that they have to ", Verb," first.") 
+    print("I've decorated my room with ", Number2, Noun2, ".")
+    print("Today I talked to a doctor and they were wearing a ", Noun3, " on their ", Part_of_the_Body_2, ".")
+    print("I heard that all doctors ", Verb, Noun4, " every day for breakfast. ")
+    print("The most ", Adjective3, " thing about being in the hospital is the ", Silly_Word, Noun, "!")
+    
+elif (n == '2'):
+    Proper_Noun = input("Person’s Name: ")
+    Noun = input("Noun: ")
+    Adjective = input("Feeling: ")
+    Adjective2 = input("Feeling2: ") 
+    Animal = input("Animal: ")
+    Verb2 = input("Verb2: ")
+    Color = input("Color: ")
+    Verb = input("Verb: ") 
+    Adverb = input("Adverb: ")
+    Number = input("Number: ")
+    Measure_of_Time = input("Measure_of_Time: ")
+    Silly_Word = input("Silly_Word: ")
+    Noun2 = input("Noun2: ")
+    print("This weekend I am going camping with ", Proper_Noun, ".")
```

Comment:
You can also using + operator for in print variables statement. But your work also work. It's good.